MODULE.bazel

```diff
@@ -1,6 +1,146 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_scala_annex")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+
+# TODO: Use buildifier instead of buildifier_prebuilt once
+# https://github.com/bazelbuild/bazel-central-registry/issues/380 is resolved
+bazel_dep(name = "buildifier_prebuilt", version = "7.3.1")
+bazel_dep(name = "rules_java", version = "7.12.2")
+bazel_dep(name = "rules_jvm_external", version = "6.5")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "stardoc", version = "0.7.1")
+
+register_toolchains(
+    "//:annex_scalafmt",
+    "//:repository_default_toolchain_21_definition",
+    "//rules/scala_proto:scalapb_scala_proto_toolchain",
+    "//src/main/scala:annex_bootstrap_2_13",
+    "//src/main/scala:annex_bootstrap_3",
+    "//src/main/scala:annex_zinc_2_13",
+    "//src/main/scala:annex_zinc_3",
+)
+
+# Please ensure these stay up-to-date with the versions in `rules/scala/versions.bzl`.
```

Comment:
Can we update this to ask folks to keep these in sync with versions.bzl and the test MODULE.bazel?

Reply:
Similar response.